api/stream.go

```diff
@@ -97,22 +97,15 @@ func PostServiceStream(c *gin.Context) {
 	//
 	// https://pkg.go.dev/github.com/sirupsen/logrus?tab=doc#Entry.WithFields
 	logger := logrus.WithFields(logrus.Fields{
-		"build":   b.GetNumber(),
 		"org":     o,
 		"repo":    r.GetName(),
+		"build":   b.GetNumber(),
```

Comment:
Reordering these doesn't really matter.

It helps me to see them in api path order so that I can make sure I didn't forget one: `.../repos/:org/:repo/build/:build/...`

I guess it is in alphabetical order now, so I can revert this if alphabetical order is preferable.